studio/runner.py

```diff
@@ -313,7 +313,7 @@ def submit_experiments(
 
             queue_name = 'pubsub_' + str(uuid.uuid4())
 
-            queue = PubsubQueue(queue_name, verbose=verbose)
+            queue = PubsubQueue(queue_name, config['database']['projectId'], verbose=verbose)
```

Comment:
now that I think about it - we should probably rather read projectId not from config, but from file to which GOOGLE_APPLICATION_CREDENTIALS points. Rationale is this - people might want to use central database (e.g. with project id studio-ed756), but their own queue. In fact, the way it is set up right now it won't work for anyone who is does not have admin rights or right credentials for studio-ed756 project

Reply:
fine with me for now